samples/client/petstore/typescript-angular-v2/default/model/ApiResponse.ts

```diff
@@ -17,9 +17,8 @@
  */
 export interface ApiResponse {
     code?: number;
-
     type?: string;
-
     message?: string;
-
 }
+
```

Comment:
But do you know where these two lines come from? Why two?

Reply:
dangit.... I think I accidentally added another one after the simple eol.  I can go back and change that lol

Reply:
turns out it wasn't me.  It was there incorrectly from the enum generator

Reply:
@kenisteward ah I see. I'll create a PR to get rid of them eventually, unless anybody sees any value in having two empty lines at the end.